gap/SkeletalFinSetsForCAP.gi

```diff
@@ -21,6 +21,7 @@ InstallMethod( CategoryOfSkeletalFinSets,
         category_filter := IsCategoryOfSkeletalFinSets,
         object_filter := IsSkeletalFiniteSet,
         morphism_filter := IsSkeletalFiniteSetMap,
+        category_attribute_names := [ "CategoryOfRelations" ],
```

Comment:
This should not be necessary anymore, right?

Reply:
Yes, deleted.